components/Nav/Nav.test.tsx

```diff
@@ -92,6 +129,26 @@ describe('Nav', () => {
     expect(navElement).toBeInTheDocument();
     expect(navElement).toHaveClass('hidden');
   });
+  it('it should be hidden when path is /reset-password', () => {
+    mockUsePathname.mockImplementation(() => '/recover-password');
+
+    render(<Nav />);
+
+    const navElement = screen.getByTestId('nav');
+
+    expect(navElement).toBeInTheDocument();
+    expect(navElement).toHaveClass('hidden');
```

Comment:
We could hardcode `screen.getByTestId('nav')` into lines 139 and 140 for cleaner code since we're only using it twice.

Reply:
Could do that but is it better to have them separate for legibility or condensed code? 🤔

Reply:
@choir27 @ryandotfurrer I dunno what to do now lol. Is it really cleaner? I like it for legibility. One could say legibility is clean code?